Jenkinsfile

```diff
@@ -71,7 +80,7 @@ pipeline {
                     }
                     steps {
                         echo 'Beginning tests...'
-                        bat 'C:\\Users\\root\\.cargo\\bin\\cargo test --all'
+                        bat 'C:\\Users\\root\\.cargo\\bin\\cargo test --all --features "vulkan sdl_controller json saveload"'
```

Comment:
SDL2 dependency seems to be missing in our Windows setup.

Reply:
@fhaynes is this a possibility to get running, or should've disable the feature on windows?

Reply:
Ah, your watching closely @torkleyy . Yah, I just removed the SDL requirement. CI went swimmingly now.